modules/light-clients/08-wasm/types/contract_api.go

```diff
@@ -28,6 +28,7 @@ type QueryMsg struct {
 type StatusMsg struct{}
 
 // ExportMetadataMsg is a queryMsg sent to the contract to query the exported metadata of the wasm client.
+// DEPRECATED: core IBC will export all metadata.
```

Comment:
linter complaino, I think go deprecation warnings are just standard casing

```suggestion
// Deprecated: core IBC will export all metadata.
```